toqito/matrix_props/tests/test_is_pseudo_unitary.py

```diff
@@ -0,0 +1,44 @@
+"""Test is_pseudo_unitary."""
+
+import numpy as np
+
+from toqito.matrix_props import is_pseudo_unitary
+from toqito.rand import random_unitary
+
+
+def test_is_unitary_random():
+    """Test that random unitary matrix with signature diag([1, 1]) return True."""
+    mat = random_unitary(2)
+    np.testing.assert_equal(is_pseudo_unitary(mat, p=2, q=0), True)
+
+
+def test_is_pseudo_unitary_lorentz_boost():
+    """Test that Lorentz boost matrix with signature diag([1, -1]) returns True."""
+    theta = np.random.rand()
+    mat = np.array([[np.cosh(theta), np.sinh(theta)], [np.sinh(theta), np.cosh(theta)]])
+    np.testing.assert_equal(is_pseudo_unitary(mat, p=1, q=1), True)
```

Comment:
This could be combined into 1 test using https://docs.pytest.org/en/stable/how-to/parametrize.html. 

The only thing that's different is the input matrix and values of p & q. But both check if the provided matrix is unitary or pseudo-unitary.